widgets/modules/EnsEMBL/Web/Apache/Static.pm

```diff
@@ -0,0 +1,19 @@
+package EnsEMBL::Web::Apache::Static;
+
+use strict;
+use warnings;
+use List::Util qw( any );
+
+my @patterns_to_skip_caching = (
+  qr/alphafold\/.*\.js$/ # javascript files in the alphafold folder
+);
+
+sub should_skip_caching {
+  my $r = shift;
+  my $uri = $r->uri;
+
+  return any { $uri =~ $_ } @patterns_to_skip_caching;
```

Comment:
i dont think you need `any` , it will return 1 if the pattern match

Reply:
I'm confused; my `@patterns_to_skip_caching` are still an array...

Reply:
we want the function to return 1 if it matches the regex, i have been trying to get this working with one line but struggling with it. Normally we can use grep for iterating through an array but in this scenario we are doing the other way round. Maybe the foreach will be cleaner. Also i dont think `any` is a perl syntax, i am confused as how this works.

Reply:
It uses a List util that is present in our installation (https://github.com/Ensembl/public-plugins/blob/7f9fbb0a27df59b6c6aae725d11449c12f13d89c/widgets/modules/EnsEMBL/Web/Apache/Static.pm#L5). It's fine, and cleaner than the foreach.

[docs](https://perldoc.perl.org/List::Util#any)

Reply:
well learn a new library now and i was always using grep :) it does what we want. i am happy with this.